app/src/main/java/com/byeboo/app/presentation/quest/review/common/personal/detail/MyDetailAnswerViewModel.kt

```
@@ -148,6 +148,9 @@ class MyDetailAnswerViewModel
             }
         }
 
+        fun onHeartClicked() {
+        }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)